### PR TITLE
Add @now/frameworks

### PR DESCRIPTION
**Pull request by shuding**
- name
- slug
- tagline
- logo (optional)
- website (optional)

### CODE REVIEW

**Review by leo — CHANGES_REQUESTED** [review]
Looks good! But we should maintain the logos in the `zeit/now` repository (inside the `@now/frameworks` package.

**Review by leo — APPROVED** [review]
There are several missing, but that's expected, as it's copied from the list of examples, not the list of detectors. Let's get it shipped so that @quietshu can wrap up the `/frameworks` API and make the frontend consume it.

File: packages/frameworks/package.json [hunk 1]
```diff
@@ -0,0 +1,6 @@
+{
+  "name": "@now/frameworks",
+  "version": "0.0.1",
+  "main": "frameworks.json",
```
Comment (styfle) [inline packages/frameworks/package.json:4]: This seems really similar to the framework detection that was added to `@now/build-utils` by @TooTallNate and @AndyBitz except this is just static files? Why?

Comment (shuding) [reply packages/frameworks/package.json:4]: @styfle we're gonna use this as a standalone lib, which will be used by both API, docs and front. cc @leo

**Pull request merged**